xfce-base/libxfce4util/metadata.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-  <maintainer type="project">
```

Comment:
What happened here? Please don't change indentation unnecessarily, and if you do, then please use one tab for indentation.

Reply:
I did differ from other files. In new commit, these lines are copy-pasted.